src/components/Select/SingleSelect.tsx

```diff
@@ -129,7 +127,7 @@ const SingleSelect = (props: TSingleSelectPropTypes): JSX.Element | null => {
           rightIconProps={{
             name: isOpen ? 'caret-up' : 'caret-down'
           }}
-          readonly={withSearch && options.length <= SELECTED_VISIBLE_MIN_COUNT}
+          readonly={options.length <= SELECTED_VISIBLE_MIN_COUNT}
```

Comment:
Does all single selects have search functional? in HRM we don't have search

Reply:
agreed with the designers